### PR TITLE
improve jupyter kernel doc

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -557,8 +562,8 @@ To provide a new name, use the same pragma and provide a name as parameter:
 If the job's name is not set, the ProActive kernel uses the current notebook name, if possible, or gives a random one.
 
 ==== List all submitted jobs
-
-To get all submitted job ids and names, use `list_submitted_jobs` pragma this way:
+s
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: s?

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -572,32 +577,40 @@ To get the job result(s), the user has to use the `#%get_result()` pragma by pro
 #%get_result(name=JOB_NAME)
 ```
 
-or by the job id:
+Or by the job ID:
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: previously u used "Or," which I approve but here u removed it. I prefer with "comma"

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 3]
```diff
@@ -572,32 +577,40 @@ To get the job result(s), the user has to use the `#%get_result()` pragma by pro
...
 
 ```python
 #%get_result(id=JOB_ID)
 ```
 
 The returned values of your final tasks will be automatically printed.
 
-==== Show ActiveEon Studio portals
+=== See and Use ActiveEon Portals directly in Jupyter
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: See => Display 
Use => use
in => from

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 1 above)
Comment (mklkun) [reply src/docs/MLOS/MLOSUserGuide.adoc]: I really don't know from where it came. xD

**Pull request merged**